barretenberg/cpp/src/barretenberg/commitment_schemes/utils/batch_mul_native.hpp

```diff
@@ -4,40 +4,48 @@
 #include <vector>
 
 namespace bb {
-template <typename Curve> class CommitmentSchemesUtils {
+/**
+ * @brief Utility for native batch multiplication of group elements
+ * @note This is used only for native verification and is not optimized for efficiency
+ */
+template <typename Commitment, typename FF>
+static Commitment batch_mul_native(const std::vector<Commitment>& _points, const std::vector<FF>& _scalars)
```

Comment:
Make utility function in a class be simply a function template (static class members useful for reducing verbosity when we have complicated types, e.g. around flavors).